src/libstd/path.rs

```diff
@@ -323,6 +323,13 @@ unsafe fn u8_slice_as_os_str(s: &[u8]) -> &OsStr {
     mem::transmute(s)
 }
 
+// Detect scheme on Redox
+#[inline]
+#[allow(unused_variables)]
```

Comment:
Oh right; inlining probably will happen anyway. The `allow` was needed with `#[cfg]` since the argument was unused on non-Redox, but is unneeded now.